src/models/chapter/index.yaml

```
@@ -0,0 +1,8 @@
+type: object
+required:
+  - index
+description: "The index for the chapter"
+properties:
+  index:
+    type: integer
+    example: 1 
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)